docs/guides/fractional-gates.ipynb

```diff
@@ -0,0 +1,353 @@
+      "text/plain": [
+       "<Figure size 1400x480 with 1 Axes>"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Draw timeline of circuit with fractional gates\n",
+    "draw_timeline(ising_circuit_fractional, idle_wires=False, time_range=(0, 500), style=IQXSimple())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "48bad5e6-3c7a-4bd0-b050-e440c3257be2",
+   "metadata": {},
+   "source": [
+    "## Where to use fractional gates\n",
+    "\n",
+    "Historically, the basis gates available on QPUs in the fleet of IBM Quantum hardware have been **`CZ`**, **`X`**, **`RZ`**, **`SX`**, and **`ID`** which creates a constraint on circuits with single and two-qubit rotations. For example, an $R_X(\\theta)$ gate, when transpiled to a QPU in the IBM Quantum fleet, must decompose into a series of $RZ$ and $\\sqrt{X}$ gates -- creating a circuit with a depth of five instead of one.\n",
```

Comment:
```suggestion
    "Historically, the basis gates available on IBM Quantum QPUs have been **`CZ`**, **`X`**, **`RZ`**, **`SX`**, and **`ID`**, which can constrain circuits with single and two-qubit rotations. For example, an $R_X(\\theta)$ gate, when transpiled, must decompose into a series of $RZ$ and $\\sqrt{X}$ gates -- creating a circuit with a depth of five instead of one.\n",
```